x-pack/plugins/uptime/public/components/synthetics/executed_step.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiSpacer, EuiText } from '@elastic/eui';
+import React, { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { CodeBlockAccordion } from './code_block_accordion';
+import { Ping } from '../../../common/runtime_types/ping';
+import { euiStyled } from '../../../../../../src/plugins/kibana_react/common';
+import { StepScreenshots } from './check_steps/step_expanded_row/step_screenshots';
+
+const CODE_BLOCK_OVERFLOW_HEIGHT = 360;
+
+interface ExecutedStepProps {
+  step: Ping;
+  index: number;
+  browserConsole?: string;
+}
+
+const Label = euiStyled.div`
+  margin-bottom: ${(props) => props.theme.eui.paddingSizes.xs};
+  font-size: ${({ theme }) => theme.eui.euiFontSizeS};
```

Comment:
Could `font-size` and `font-weight` for both Label and Message be achieved with `EuiText`? And then we would only add the CSS for the margin? Edit: not seeing a prop for bold.

Reply:
going to leave this as is for now.